codecov.yml

```diff
@@ -1,4 +1,5 @@
 codecov:
+  require_ci_to_pass: false
   ci:
     - !appveyor
```

Comment:
This seems to be the issue. We weren't able to properly parse the `yaml`, because we don't accept `!` for `codecov -> ci`

```suggestion
```

Reply:
Thanks, I'll give it a try!

Reply:
Is there an online tool or command line tool for parsing/linting the `yaml`? Or a way to access the parsing log? Or is that just internal for you for now?

Reply:
@tylerjereddy amazing 🤞 ! Currently this is internal, but we are looking to publish the validator in the very near future.

Reply:
Ah great, that would probably reduce the frequency I end up pinging you a bit :)

Reply:
Hello from the very new future, it is, in fact, possible to do this. We'll be working on getting this integrated into the uploaders, starting with the `bash` uploader. You can run 

```
curl -X POST --data-binary @codecov.yml https://codecov.io/validate
```